stories/Header/StickyHeader.js

```diff
@@ -0,0 +1,208 @@
+import React, { Component } from 'react'
+import { boolean } from '@storybook/addon-knobs/react'
+import {
+  Button,
+  Collapse,
+  Row,
+  Col,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+  Header,
+  Headers,
+  HeaderContent,
+  HeaderSearch,
+  HeaderRightZone,
+  HeaderSocialsZone,
+  HeaderToggler,
+  HeaderLinkZone,
+  Icon,
+  LinkList,
+  LinkListItem,
+  Nav,
+  NavItem,
+  NavLink,
+  HeaderBrand,
+  UncontrolledDropdown
+} from '../../src'
+
+// eslint-disable-next-line react/prop-types
+// eslint rule has to be disable for few lines here as Storybook addons will go in error
+// if PropTypes are declared in these components
+export class SlimHeader extends Component {
+  state = {
+    collapsed: true
+  }
```

Comment:
Remove this and make the Class Component a Function Component using `UncontrolledCollapse`

Reply:
I have found other components in the library that have class component. Should I try to convert them into a function component too?

Reply:
I know, but let's keep each PR focused on a single topic.